index.bs

```diff
@@ -379,16 +380,16 @@ serve as an identifier if misused/abused [[OLEJNIK-BATTERY]].
 </p>
 
 <h3 class=question id="other-data">
-  What data does this specification expose to an origin?  Please also
-  document what data is identical to data exposed by other features, in the
-  same or different contexts.
+  What data does this specification expose to an origin or other party?  
```

Comment:
I think "origin" is fine since "parties" are origins in the web.

Reply:
Other parties include the user agent vendor that I don't believe would be classified as origins. Consider a feature where the user agent vendor is capturing personal information. Considerations related to privacy would at least include;

a) consent obtained for use of that information;
b) transparency to assure the user that the consent preferences were respected; and
c) security issues associated with the way that data is stored and deleted, among others. 

Considering security and privacy beyond protocols and origins is an important theme I would like to see developed in PING and this document. What mitigations can be applied that do not relate entirely to technical standards? For example; accepting terms of service and default settings at the point of user agent setup or upgrade will enable implementors to consider solutions that are not restricted to permission pop ups that are annoying, or defaults that are so restrictive they impair functionality, or expose user agent vendors to competition issues by gaining consent across vertically integrated services at the point of account setup which is impossible for other players in the market to achieve.

I hope we can all agree that a situation where acceptable privacy can only be achieved by a very small number of implementors would not be compatible with the W3C antitrust policy or the mission and purpose of the W3C.

Reply:
The point that the user agent (vendor) is a party outside of "origins" - that effectively has "root" -  is a good one. Another example is "geolocation" where information is sent out out band to the browser maker as part of the API.